judge/views/problem.py

```diff
@@ -443,8 +443,9 @@ def get_noui_slider_points(self):
         if not points:
             return 0, 0, {}
         if len(points) == 1:
-            return points[0], points[0], {
+            return points[0] - 1, points[0] + 1, {
```

Comment:
Why does this fix things?

Reply:
If the problem list only has 5p problems, the server will give `point_start=5` and `point_values.min=4`. This runs into trouble with

https://github.com/DMOJ/online-judge/blob/97f2c450e88814012d19cbcfbec3ae74d14415cb/templates/problem/list.html#L121

Disabled inputs aren't put in the querystring.

Same goes for `point_end=5` and `point_values.max=6`.